.github/workflows/publish_generate_mvt.yml

```diff
@@ -49,24 +60,27 @@ jobs:
           AWS_SECRET_ACCESS_KEY: "op://Data Engineering/DO_keys/AWS_SECRET_ACCESS_KEY"
           AWS_ACCESS_KEY_ID: "op://Data Engineering/DO_keys/AWS_ACCESS_KEY_ID"
 
-      - name: Download file
+      - name: Download source file
         run: |
           python3 -m dcpy.connectors.edm.publishing download_file \
-            -p db-${{ inputs.product }} \
-            -v ${{ inputs.version }} \
-            -f ${{ inputs.filepath_to_shapefile }}
+            --product db-${{ inputs.product }} \
+            --version ${{ inputs.version }} \
+            --filepath ${{ inputs.filepath_to_shapefile }}
 
       - name: Generate MVT
         run: | 
           python3 -m dcpy.utils.geospatial \
-            -i $(basename ${{ inputs.filepath_to_shapefile }}) \
-            -o ${{ inputs.product }}_mvt
+            --product ${{ inputs.product }} \
+            --input-path $(basename ${{ inputs.filepath_to_shapefile }}) \
+            --min-zoom ${{ inputs.min_zoom }} \
+            --max-zoom ${{ inputs.max_zoom }}
 
-      - name: Publish MVT
+      - name: Upload MVT
         run: | 
           python3 -m dcpy.utils.s3 \
-            -b de-sandbox \
+            --bucket de-sandbox \
             --folder-path ${{ inputs.product }}_mvt \
             --s3-path mvt/${{ inputs.product }}/${{inputs.version}}/ \
             --acl private \
+            --max-files 800000 \
```

Comment:
Was this number reached by testing

Reply:
looks like [this run](https://github.com/NYCPlanning/data-engineering/actions/runs/6395608302/job/17359648274#step:6:68) hit 617,457 when using zoom levels of 15-20

Reply:
can we zip these up? Or do they need to be separate on s3?

Reply:
Bc we're eventually just dumping them all into PostGIS, right?

Reply:
that sounds right. but I don't think they've setup a postigs DB yet or chosen an ideal file format they'd like for the current prototyping

also the `gdal.VectorTranslate` call that generates the tiles could even just have a DB as a destination (although there may be a desire to archive all versions as files)